### PR TITLE
Old branch script reputations add on

### PR DESCRIPTION
**Pull request by roysagi**
## Status
- [x] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
related: https://github.com/demisto/etc/issues/24383

## Description
adding reputations.json support for feature branch script

### CODE REVIEW

File: Utils/old_content_branch.py [hunk 1]
```diff
@@ -190,6 +206,21 @@ def edit_all_packs(new_to_version):
...
+def edit_reputations_json(new_to_version):
+    print("Updating reputations.json\n")
+    rep_json_path = "Packs/NonSupported/IndicatorTypes/reputations.json"
+    with open(rep_json_path, 'r') as f:
+        rep_content = ujson.load(f)
+
+    for reputation in rep_content.get('reputations', []):
+        if parse_version(reputation.get('toVersion', "99.99.99")) > parse_version(new_to_version) > \
+                parse_version(reputation.get("fromVersion", "0.0.0")):
+            reputation['toVersion'] = new_to_version
```
Comment (ronykoz) [inline Utils/old_content_branch.py:218]: are we adding the fromversion of this in the content creator? i don't think it's needed here @yaakovi WDYT?

Comment (yaakovi) [reply Utils/old_content_branch.py:218]: you can ignore the `fromversion`.
this can be relevant only for `toversion` lower than 4.0.0

**Comment by ronykoz** [comment]
@barchen1 can you take care of the last comment?

File: Utils/old_content_branch.py (hunk 1 above)
Comment (barchen1) [reply Utils/old_content_branch.py]: done

**Pull request merged**